### PR TITLE
Region Show Page

### PR DESCRIPTION
**Pull request by KettyLezama**
+ Created RegionShowContainer with fetch request from "api/v1/regions_controller"
+ Created RegionTile
+ Edited "api/v1/regions_controller" to have "def show" method
+ Edited routes.rb
+ Updated seeds file with flavor objects
+ Yarn example test added
+ Rspec controller test added and passed

### CODE REVIEW

File: app/javascript/react/components/RegionShowContainer.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import FlavorTile from "./FlavorTile";
+
+const RegionShowContainer = (props) => {
+  const [getFlavors, setFlavors] = useState([]);
+  useEffect(() => {
```
Comment (safang99) [inline app/javascript/react/components/RegionShowContainer.js:6]: add a line between 5 & 6 for readability

File: app/javascript/react/components/RegionShowContainer.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      .then((body) => {
+        let flavorList = body;
+        setFlavors(flavorList);
+      })
+      .catch((error) => console.error(`Error in fetch: ${error.message}`));
+  }, []);
+  const listFlavors = getFlavors.map((flavor) => {
+
+    return <FlavorTile key={flavor.name} name={flavor.name} />;
+  });
```
Comment (safang99) [inline app/javascript/react/components/RegionShowContainer.js:28]: add a line between 24 & 25 and remove the empty line on 26

File: app/views/homes/index.html.erb [hunk 3]
```diff
@@ -2,4 +2,6 @@
 <p>
   Modify me in `app/views/homes/index.html.erb` or change the root directive in
   `config/routes.rb`
+
+  This is our static page
 </p>
```
Comment (safang99) [inline app/views/homes/index.html.erb:7]: This file should only be whats on line 1 ideally.  We can remove 2-7

File: app/controllers/api/v1/regions_controller.rb [hunk 4]
```diff
@@ -1,5 +1,11 @@
...
-    end
+  def index
+    render json: Region.all.order('name')
+  end
+
+  def show
+    @region = Region.find(params[:id])
+    @flavors = @region.flavors
+    render json: @flavors.order('name')
+  end
```
Comment (safang99) [inline app/controllers/api/v1/regions_controller.rb:10]: Does a region show page not display the name of the region? It looks like its just returning the associated flavors currently.  We can make use of `serializers` and then we can simply return,
```ruby
def show
  render json: Region.find(params[:id])
end
```

and on the serializer we just have to call `has_many :flavors`

File: app/javascript/react/components/RegionShowContainer.js (hunk 1 above)
Comment (KettyLezama) [reply app/javascript/react/components/RegionShowContainer.js:6]: Added line.

File: app/javascript/react/components/RegionShowContainer.js (hunk 2 above)
Comment (KettyLezama) [reply app/javascript/react/components/RegionShowContainer.js:28]: Complete.

File: app/views/homes/index.html.erb (hunk 3 above)
Comment (KettyLezama) [reply app/views/homes/index.html.erb]: This was resolved in merge conflict resolution.

**Comment by RobPerez155** [comment]
Added serializers and updated feature

File: app/controllers/api/v1/regions_controller.rb (hunk 4 above)
Comment (RobPerez155) [reply app/controllers/api/v1/regions_controller.rb:8]: Added serializers and updated related files

**Pull request merged**